src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsQuestionDetails.java

```diff
@@ -8,6 +8,8 @@
 import java.util.Map;
```

Comment:
See the discussion on `FeedbackRankOptionsQuestionDetails` and also let's name these as parameters to be consistent.